pages/1.11/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -629,120 +587,134 @@ In this step, a non-native Marathon instance is installed on DC/OS with the Meso
             "port":0,
             "name":"libprocess"
          }
-      ],
-      "fetch":[  
-         {  
-            "uri":"file:///home/<linux-user>/docker.tar.gz"
-         }
       ]
     }
     ```
 
-1.  Deploy the Marathon instance.
+1.  Deploy the Marathon instance, passing the JSON file you created in the previous step.
 
     ```bash
-    dcos marathon app add config.json
+    dcos marathon app add marathon-filled.json
     ```
 
-# Step 8 - Grant User Access to Non-Native Marathon
-In this step, a user is granted access to the non-native Marathon instance.
+# Step 7: Grant User Access to Non-Native Marathon
+By now, your new Marathon instance is accessible only by the DC/OS Superusers. In order to give access to regular users, you need to explicitly given them access permissions, according to your cluster security mode: 
 
-1. Log into the DC/OS GUI as a user with the `superuser` permission.
+  ## {.tabs}
 
-   ![Login](/1.11/img/gui-installer-login-ee.gif)
+  ### Disabled
 
-1.  Select **Organization** and choose **Users** or **Groups**.
+  For each `${USER_ACCOUNT}` that you want to give access, if you are running a cluster in `disabled` security mode, do:
 
-1.  Select the name of the user or group to grant the permission to.
+  ```bash
+  # Access to the new marathon service under `/service/<name>` URL
+  dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:service:${MARATHON_INSTANCE_NAME} full
+  dcos security org users grant ${USER_ACCOUNT} dcos:service:marathon:${MARATHON_INSTANCE_NAME}:services:/ full
 
-    ![Add permission cory](/1.11/img/services-tab-user.png)
+  # Access to the Mesos tasks and their sandbox
+  dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:ops:mesos full
+  dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:ops:slave full
 
-1.  From the **Permissions** tab click **ADD PERMISSION**.
+  # (Optionally) Access to the marathon instance that runs on the root 
+  # Marathon and can be controlled via the DC/OS UI
+  dcos security org users grant ${USER_ACCOUNT} dcos:adminrouter:service:marathon full
+  ```
 
-1.  Click **INSERT PERMISSION STRING** to toggle the dialog.
+  ### Permissive
```

Comment:
Arrange this into tabs maybe like you did above?

Reply:
It is tabbed. Unless you are referring to the `full/individual` sub-content?